plugins/arOidcPlugin/lib/oidcUser.class.php

```diff
@@ -43,16 +43,27 @@ public function initialize(sfEventDispatcher $dispatcher, sfStorage $storage, $o
      *
      * @param null|mixed $username
      * @param null|mixed $password
-     *
-     * @return bool
      */
-    public function authenticate($username = null, $password = null)
+    public function authenticate($username = null, $password = null): bool
     {
         $authenticated = false;
         $user = null;
         $authenticateResult = false;
         $email = null;
 
+        $this->logger->err(sprintf('%s', json_encode(sfConfig::getAll())));
```

Comment:
Was this a debug message?

Reply:
Yes it is! :laughing: :facepalm: 

Thanks for catching this! :+1:

Reply:
Removed - see latest commit